src/client/pages/Console/components/dashboard/AdaptiveAnimation.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+
+// If you don't have a version of React that supports
+// hooks, you can use a class-based version of this
+// component in ProgressProviderUsingClass.js
+const ProgressProvider = ({ valueStart, valueEnd, children }) => {
+  const [value, setValue] = React.useState(valueStart)
```

Comment:
nit: If you could choose a more descriptive name, i think that would go a long way for maintainance.